pkg/network/tracer/tracer_classification_test.go

```diff
@@ -672,6 +675,88 @@ func testProtocolClassification(t *testing.T, cfg *config.Config, clientHost, ta
 			},
 			validation: validateProtocolConnection,
 		},
+		{
```

Comment:
add tests for db creation, update, insert, delete
any command that we classify on